flink-staging/flink-ml/src/main/scala/org/apache/flink/ml/classification/SVM.scala

```diff
@@ -242,8 +297,17 @@ object SVM{
         }
       }
 
-      override def predict(value: T, model: DenseVector): Double = {
-        value.asBreeze dot model.asBreeze
+      override def predict(value: T, model: DenseVector):
+        Double = {
+
+        val rawValue = value.asBreeze dot model.asBreeze
+
+        if (outputDecisionFunction) {
+          rawValue
+        }
+        else {
```

Comment:
Actually I meant the else statement.

'''
if (outputDecisionFunction) {
  rawValue
} else {
  ...
}
'''